wireshark/README.md

```diff
@@ -0,0 +1,11 @@
+This directory contains a Dockerfile that builds wireshark that is patched to understand the OIDs and codepoints in TLS 1.3 that are supported by OQS-OpenSSL.
+
+## Quick start
+
+1) Be sure to have [docker installed](https://docs.docker.com/install).
+2) Run `docker build -t openquantumsafe/wireshark .` to create an QSC-enabled (codepoint and OID aware) wireshark docker image.
+
+## Usage
+
+Information how to use the image is [available in the separate file USAGE.md](USAGE.md).
+
```

Comment:
```suggestion

## Contributors

- Michael Baentsch
- Anthony Hu (wolfSSL)
```

Reply:
We so far only have a top-level contribution list (to which I added Anthony now) and think we should leave it at that in the interest of short README's if that's OK for you.